tests/util/__snapshots__/calendar-helper.spec.ts.snap

```diff
@@ -0,0 +1,94 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Don't need snapshots for functions test

Reply:
Understand. We seldom do this. But if your intention is to "freeze" the expected outputs, then I guess it is fine.